js/http.js

```diff
@@ -123,7 +123,8 @@ var ERMrest = (function (module) {
                         deferred.resolve(response);
                     },
                     function(response) {
-                        if (response.status in _retriable_error_codes && count < max_retries) {
+                        var status = response.status || response.statusCode;
```

Comment:
@robes Angular doesn't uses that but in General Node.js Http module returns *statusCode*  in response. This is the reason I added a fallback mechanism.